src/WebJobs.Script.Grpc/Channel/GrpcWorkerChannel.cs

```diff
@@ -1068,6 +1068,8 @@ private async Task<object> GetBindingDataAsync(ParameterBinding binding, string
                 case ParameterBindingType.Data:
                     // Data was transferred by the worker using RPC
                     return binding.Data.ToObject();
+                case ParameterBindingType.None:
+                    return null;
                 default:
                     throw new InvalidOperationException("Unknown ParameterBindingType");
```

Comment:
I know this isn't your change... but can you update this exception to include the type in case we ever see this?